src/test/java/com/atlassian/db/replica/api/AuroraClusterDiscoveryTest.java

```diff
@@ -6,21 +6,7 @@
 import org.junit.jupiter.api.Test;
 import org.mockito.Mockito;
 
-import java.sql.Array;
-import java.sql.Blob;
-import java.sql.CallableStatement;
-import java.sql.Clob;
-import java.sql.Connection;
-import java.sql.DatabaseMetaData;
-import java.sql.NClob;
-import java.sql.PreparedStatement;
-import java.sql.SQLClientInfoException;
-import java.sql.SQLException;
-import java.sql.SQLWarning;
-import java.sql.SQLXML;
-import java.sql.Savepoint;
-import java.sql.Statement;
-import java.sql.Struct;
+import java.sql.*;
```

Comment:
that was not intended, I will revert this and make sure refactoring in my ide does not do this in the future